### PR TITLE
Devfix before master

### PR DESCRIPTION
**Pull request by RaphaelBS-WCS**
fixing dev:
- liens admin_navbar dysfonctionnels dans form add et edit 
(en dehors de la page principale)
- correction des fonctions crud de services

### CODE REVIEW

**Review by khadimb — APPROVED** [review]
bien

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -19,28 +19,26 @@ public function index(): string
...
             // clean $_POST data
             $services = array_map('trim', $_POST);
-
             $servicesManager->update($services);
-            header('Location: Admin/index');
+            header('Location: /Admin/index');
         }
         return $this->twig->render('Admin/Services/edit_service.html.twig', [
             'services' => $services
             ]);
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php:35]: indentation

File: src/View/Admin/Home/Component/_admin_navbar.html.twig [hunk 2]
```diff
@@ -12,8 +12,8 @@
         <div class="collapse navbar-collapse justify-content-end" id="navbarNavAltMarkup">
             <div class="navbar-nav">
                 <a class="nav-link" href="/">ACCUEIL VISITEUR</a>
-                <a class="nav-link" href="#services">PRESTATIONS</a>
-                <a class="nav-link" href="#testimonies">TEMOIGNAGES</a>
+                <a class="nav-link" href="/Admin/index/#adminService">PRESTATIONS</a>
+                <a class="nav-link" href="/Admin/index/#adminTestimonies">TEMOIGNAGES</a>
```
Comment (LucyMerlier) [inline src/View/Admin/Home/Component/_admin_navbar.html.twig:16]: you dont need full route to go to an anchor, just #adminService and #adminTestimonies will do

File: src/View/Admin/Home/Component/_form_service.html.twig [hunk 3]
```diff
@@ -9,14 +7,14 @@
...
         <label for="price1hour">Tarif 1h :</label>
-        <input class="price1hour" type="number" name="price1hour" placeholder="70" value="{{ service.price1hour }}" required>
+        <input class="price1hour" type="number" name="price1hour" placeholder="70" value="{{ services.price1hour }}" required>
         <label for="price30min">Tarif 30min :</label>
-        <input class="price30min" type="number" name="price30min" placeholder="35" value="{{ service.price30min }}"required>
+        <input class="price30min" type="number" name="price30min" placeholder="35" value="{{ services.price30min }}">
     <div>
         <label for="description">Votre message :</label>
-        <textarea name="description" id="description" rows="4" placeholder="Description du service" value="{{ service.description }}"required></textarea>
+        <textarea name="description" id="description" rows="4" placeholder="Description du services" required>{{ services.description }}</textarea>
```
Comment (LucyMerlier) [inline src/View/Admin/Home/Component/_form_service.html.twig:17]: "description du services" => service singular

File: src/View/Admin/Home/Component/_form_service.html.twig [hunk 4]
```diff
@@ -1,5 +1,3 @@
-<form action="/Admin/addservice" class="form" method= "POST">
```
Comment (LucyMerlier) [inline src/View/Admin/Home/Component/_form_service.html.twig:1]: wait... you deleted the opening form tag?...

File: src/View/Admin/Services/add_service.html.twig [hunk 5]
```diff
@@ -6,5 +6,7 @@
 {% endblock %}
 
 {% block content %}
+    <form class="form" method= "POST">
```
Comment (LucyMerlier) [inline src/View/Admin/Services/add_service.html.twig:9]: ... why is the opening form tag here???

File: src/View/Admin/Services/edit_service.html.twig [hunk 6]
```diff
@@ -6,5 +6,7 @@
 {% endblock %}
 
 {% block content %}
+    <form class="form" method= "POST">
+        <h2 class="add_service">Editer une prestation</h2>
```
Comment (LucyMerlier) [inline src/View/Admin/Services/edit_service.html.twig:10]: oh ok I get what you where trying to do. don't to it like that, just put your h2 outside your form and your form tag back into your form component

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/Controller/AdminController.php (hunk 1 above)
Comment (RaphaelBS-WCS) [reply src/Controller/AdminController.php:35]: ok

File: src/View/Admin/Home/Component/_admin_navbar.html.twig (hunk 2 above)
Comment (RaphaelBS-WCS) [reply src/View/Admin/Home/Component/_admin_navbar.html.twig:16]: ok

File: src/View/Admin/Home/Component/_form_service.html.twig (hunk 3 above)
Comment (RaphaelBS-WCS) [reply src/View/Admin/Home/Component/_form_service.html.twig:17]: ok

File: src/View/Admin/Services/add_service.html.twig (hunk 5 above)
Comment (RaphaelBS-WCS) [reply src/View/Admin/Services/add_service.html.twig:9]: solved

File: src/View/Admin/Services/edit_service.html.twig (hunk 6 above)
Comment (RaphaelBS-WCS) [reply src/View/Admin/Services/edit_service.html.twig:10]: ok

File: src/View/Admin/Home/Component/_form_service.html.twig (hunk 4 above)
Comment (RaphaelBS-WCS) [reply src/View/Admin/Home/Component/_form_service.html.twig:1]: yes

**Review by khadimb — APPROVED** [review]
nickel

File: src/Controller/AdminController.php (hunk 1 above)
Comment (LucyMerlier) [reply src/Controller/AdminController.php:35]: not resolved

File: src/View/Admin/Home/Component/_form_service.html.twig [hunk 7]
```diff
@@ -9,14 +8,14 @@
...
         <label for="name">Nom de la prestation :</label>
-        <input class="name" type="text" name="name" placeholder="Nom de la prestation" value="{{ service.name }}"required>
+        <input class="name" type="text" name="name" placeholder="Nom de la prestation" value="{{ services.name }}"required>
         <label for="price1hour">Tarif 1h :</label>
-        <input class="price1hour" type="number" name="price1hour" placeholder="70" value="{{ service.price1hour }}" required>
+        <input class="price1hour" type="number" name="price1hour" placeholder="70" value="{{ services.price1hour }}" required>
         <label for="price30min">Tarif 30min :</label>
-        <input class="price30min" type="number" name="price30min" placeholder="35" value="{{ service.price30min }}"required>
+        <input class="price30min" type="number" name="price30min" placeholder="35" value="{{ services.price30min }}">
     <div>
```
Comment (LucyMerlier) [inline src/View/Admin/Home/Component/_form_service.html.twig:16]: missing div closing tag or stray div opening tag

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/Controller/AdminController.php (hunk 1 above)
Comment (RaphaelBS-WCS) [reply src/Controller/AdminController.php:35]: Sorry, it's done now.

File: src/View/Admin/Home/Component/_form_service.html.twig (hunk 7 above)
Comment (RaphaelBS-WCS) [reply src/View/Admin/Home/Component/_form_service.html.twig:16]: ok

**Review by Erdnaxela45 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**